airflow/cli/cli_parser.py

```diff
@@ -224,6 +224,16 @@ def _check(value):
     default=ColorMode.AUTO,
 )
 
+# DB args
+ARG_VERSION_RANGE = Arg(
+    ("-v", "--version-range"), help="Version range(start:end) for offline sql generation", default=None
+)
+ARG_REVISION_RANGE = Arg(
+    ('-r', '--revision-range'),
```

Comment:
This one is probably not going to be used very common, so we can remove the short opt here
```suggestion
    ('--revision-range',),
```